Zend/zend_ast.c

```diff
@@ -1158,6 +1159,7 @@ static void* ZEND_FASTCALL zend_ast_tree_copy(zend_ast *ast, void *buf)
 		new->attr = old->attr;
 		new->lineno = old->lineno;
 		new->op_array = old->op_array;
+		function_add_ref((zend_function *)new->op_array);
```

Comment:
With this line:

- Without OPcache: Works.
- With OPcache: Leaks.

Without this line:

- Without OPcache: Crashes.
- With OPcache: Works.

But I don't know in which location I need to insert the missing `function_add_ref()` or `destroy_op_array()` to make both work.